integration/container/windows_isolation_test.go

```diff
@@ -0,0 +1,422 @@
+package container
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	containertypes "github.com/moby/moby/api/types/container"
+	"github.com/moby/moby/api/types/mount"
+	"github.com/moby/moby/client"
+	"github.com/moby/moby/v2/integration/internal/container"
+	"github.com/moby/moby/v2/internal/testutil"
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
+	"gotest.tools/v3/skip"
+)
+
+// TestWindowsProcessIsolation validates process isolation on Windows.
+func TestWindowsProcessIsolation(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "windows")
```

Comment:
The other platform-specific test files in this package are named `*_windows_test.go` and `*_linux_test.go`, <a href="https://pkg.go.dev/cmd/go#hdr-Build_constraints:~:text=If%20a%20file%27s,in%20the%20file).">which constrains them to target only the named OS.</a> Rename the files in this PR for consistency, which will also make these `skip.If` calls redundant.